script.js

```diff
@@ -1,4 +1,122 @@
+/*
+- player 1 rolls two dice and choose the order of the dice
+- player 2 rolls two dice and choose the order of the dice
+- compare player 1 and player 2 number, state the winner
+- reset the game
+*/
+
+var gameState = "player1Roll";
+var player1RollRecords = [];
+var player2RollRecords = [];
+var player1PointsRecord = [];
+var player2PointsRecord = [];
+
+// roll dice function
+var rollDice = function () {
+  let randomDiceNum = Math.floor(Math.random() * 6) + 1;
+  //console.log(randomDiceNum);
+  return randomDiceNum;
+};
```

Comment:
good that you commented out your console.log before submission but usually we'll remove it altogether, this will allow you to collapse line 16-18 to just one single line

return Math.floor(Math.random() * 6) + 1;